components/molecule/rating/src/index.scss

```diff
@@ -19,10 +19,20 @@ $bg-molecule-rating-hover: $c-primary-light-5 !default;
 
 $m-molecule-rating-label: 0 0 0 $m-m !default;
 
+$c-molecule-rating-svg: $c-system !default;
+
 $base-class: '.sui-MoleculeRating';
 $class-star: '#{$base-class}-Star';
 $class-label: '#{$base-class}-label';
 
+%c-molecule-rating-svg svg {
+  fill: $c-molecule-rating-svg;
+}
+
+#{$class-star} {
+  @extend %c-molecule-rating-svg;
```

Comment:
I think it's not worth it using `extend` in this case specially with a name that collides with the variable name.

I find more readable:
```
#{$class-star} svg {
  fill: $c-molecule-rating-svg;
}
```

or 

```
#{$class-star} {
  svg {
    fill: $c-molecule-rating-svg;
  }
}
```